.github/workflows/tests-smoke.yaml

```
@@ -2,11 +2,21 @@ name: Smoke Test
 
 # Any change in triggers needs to be reflected in the concurrency group.
 on:
-  pull_request: {}
+  pull_request:
+    paths-ignore:
+      - '*.md'
+      - '.github/renovate.json5'
+      - 'Documentation/**'
+      - 'test/**'
   push:
     branches:
       - main
       - ft/main/**
+    paths-ignore:
+      - '*.md'
```

nit: Could add `rst` as well to catch README.
```suggestion
      - '*.md'
      - '*.rst'
```